src/components/Text/Text.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import style from './Text.st.css';
+
+export interface TextProps {
+  typography?: TYPOGRAPHY;
+  dataHook?: string;
+}
+
+export enum TYPOGRAPHY {
+  smallTitle = 'smallTitle',
+  runningText = 'runningText'
+}
+
+function getTagNameByTypography(typography: string) {
+  if (typography === TYPOGRAPHY.smallTitle) {
+    return 'h5';
+  } else {
+    return 'p';
+  }
+}
+
+const CoreText = ({typography, children, dataHook, ...rest}) => {
+  const tagName = getTagNameByTypography(typography);
+
+  return React.createElement(
+    tagName,
+    {
+      ...style(
+        'root',
+        {
+          typography
+        }
+      ),
+      'data-hook': dataHook,
+      ...rest
+    },
+    (<span data-hook="style-override-wrapper" className={style.overrideStyle}>{children}</span>),
```

Comment:
@shlomitc to apply the override the default style and display the default at the same time
for example if i want to override only the color but use the default font.
because we use style-processor the result of it is var(--something) and even if it's not a valid value the css rule not using the default but a different wrapper so we basically simulate the 'different wrapper' as the default values for color and font

Reply:
this is the override strategy that was implemented for Input